src/main/java/duke/commands/Task.java

```diff
@@ -0,0 +1,27 @@
+package duke.commands;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Consider identifying these unicodes using a constant or enum instead.

So that it is easier for people to understand what these values holds (instead of relying on the comment), as well as allowing you to use it in more location of the codebase.

Reply:
Ok, I replaced them with a constant instead